submit_cycle.sh

```diff
@@ -81,28 +97,8 @@ mkdir ${WORKDIR}
 ############################
 # create the jedi yaml name 
 
-if [[ $do_DA == "YES" || $do_hofx == "YES" ]]; then  # do DA
+if [[ $do_DA == "hofx" || $do_DA == "LETKF-OI" || $do_DA == "LETKF" ]]; then  # do DA
```

Comment:
Please change this back to do_DA being YES or NO, with a separate DAtype variable.